.storybook/preview.tsx

```diff
@@ -80,6 +80,7 @@ export const parameters: Preview['parameters'] = {
     ],
   },
   badgesConfig: {
+    ...createComponentVersion('3.1'),
```

Comment:
Each documentation page can have badges showing the version of the theme/component API. This adds an entry so we can reference 3.1 versions